lib/new_relic/agent/instrumentation/action_controller_other_subscriber.rb

```diff
@@ -0,0 +1,46 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'new_relic/agent/instrumentation/notifications_subscriber'
+require 'new_relic/agent/instrumentation/ignore_actions'
+require 'new_relic/agent/parameter_filtering'
+
+module NewRelic
+  module Agent
+    module Instrumentation
+      class ActionControllerOtherSubscriber < NotificationsSubscriber
+        def start_segment(name, id, payload)
+          segment = Tracer.start_segment(name: metric_name(name, payload))
+
+          add_segment_params(segment, payload, name)
+          push_segment(id, segment)
+        end
+
+        def add_segment_params(segment, payload, name)
+          segment.params[:filter] = payload[:filter] if payload[:filter]
+          segment.params[:keys] = payload[:keys] if payload[:keys]
+          segment.params[:original_path] = payload[:request].original_fullpath if payload[:request]
+
+          if payload[:context]
+            segment.params[:action] = payload[:context][:action]
+            segment.params[:controller] = payload[:context][:controller].to_s
```

Comment:
Is there a way to get at the string value that is being provided for the `:controller` payload key [here](https://github.com/rails/rails/blob/fc950324bdbca393b29f1c26d2bf4e7bb2aa8c42/actionpack/lib/action_controller/metal/instrumentation.rb#L59)?

If not, and we're left working with an actual controller class here, I suggest using `name` instead of `to_s` to better convey that the payload value is a class.

Reply:
What you linked to is the payload you get for process_action, which is handled by the original subscriber class already. Unfortunately, these payloads don't have the same stuff in them. 
Also, it should already be a string at this point, so .name wouldn't work, but calling to_s is redundant, so i just removed it instead. 
fbccb6e6d302c3e42a88e15f3a556399944a795b